### ISSUE TITLE
Need any help?

### ISSUE THREAD

**Issue by TomerAberbach** [dagrejs/dagre-d3#410]
I've been using this library and I've noticed there are a decent number of issues and pull requests without responses. I know life happens and repositories can become difficult to maintain.

Do you need any help triaging issues, reviewing PRs, etc.? I would be happy to contribute!

Thanks for this awesome library! :smile:

**Comment by TomerAberbach** [comment]
Also happy to expand that offer to include any repository in the @dagrejs org

**Comment by hachtman** [comment]
We are also about to be making use of this library in prod @TomerAberbach and would be happy to help from Aug. This is indeed an excellent lib and we'd be keen to help.

**Comment by its-a-feature** [comment]
If either of you two are still offering help, there's only one thing I can't seem to get working - getting a parent node to be large enough to fit its label.
![image](https://user-images.githubusercontent.com/5313261/123898609-7a599380-d91a-11eb-85c1-ba1937ec6c1e.png)
I'm trying to cluster my elements by certain properties, but the clusters don't seem to take labels into account either for the cluster label or for any of the inner node labels. Any suggestions?

**Comment by Maybe-nextime** [comment]
hello,I'm trying change path style when drawing;e.g Right angle polyline,can you help me?

**Comment by timkpaine** [comment]
@lutzroeder @cpettitt im happy to become a maintainer as well, I use this library in [ipydagred3](https://github.com/timkpaine/ipydagred3)

**Comment by tbo47** [comment]
just fyi https://github.com/tbo47/dagre-es @timkpaine

**Comment by timkpaine** [comment]
@tbo47 im happy to be a contributor anywhere 😂

**Comment by rienheuver** [comment]
Apparently this package has been set as deprecated now, and actually so has https://github.com/dagrejs/dagre itself. Is there any commonly accepted alternative? The fork by @tbo47 doesn't seem very active either, or is it? I'm currently looking into the cytoscape-version https://github.com/cytoscape/cytoscape.js-dagre
But I think I'd rather use D3 for make this than Cytoscape

**Comment by timkpaine** [comment]
I've been trying to pickup some maintainer work, but I do not have maintainer access yet.

**Comment by tbo47** [comment]
you can submit a PR and i will review it
https://github.com/tbo47/dagre-es/pulls?q=is%3Apr+is%3Aclosed

**Comment by rienheuver** [comment]
Continuation of this repo would be preferred in my opinion, but we would need to get response from @dagrejs. Maybe @cpettitt can inform us further?

**Comment by tbo47** [comment]
yep, I would prefer if the project were still maintained here but it has not. And for a long time now.

**Comment by timkpaine** [comment]
I'd like to become a maintainer of all 4 of these projects so I can make the changes I want for my library: https://github.com/timkpaine/ipydagred3

I'm happy to donate my library to this org too. I tried to ping existing admins and haven't gotten a response, im hesitant to fork because projects often have a way of turning back on (see the one here marked "starting to be supported again" as an example).

**Comment by smetroid** [comment]
I can also help anyway I can, I am using these libraries for a personal prof of concept project.
